src/app/campaigns/campaign-edit/campaign-edit.component.html

```diff
@@ -38,6 +38,33 @@ <h1 class="my-2 text-2xl uppercase">
           <input id="photo" type="file" formControlName="photo" multiple>
         </div>
 
+        <div class="mt-2">
+          <label class="block">Return frequency</label>
+          <select id="returnFrequency" formControlName="returnFrequency">
+            <option value="">-- None --</option>
+            <option *ngFor="let item of ReturnFrequencies" value="{{ item }}">{{ item | titlecase }}</option>
+          </select>
+        </div>
+
+        <div class="mt-2">
+          <label class="block">Return value</label>
+
+          <fieldset class="contents" [disabled]="isCampaignReturnValueDisabled()">
+            <label class="pr-2">From:</label>
+            <input class="disabled:bg-gray-200 disabled:cursor-not-allowed" appPercentageMask
+                   id="returnFrom" type="text" formControlName="returnFrom">
+          </fieldset>
+
+          <fieldset class="contents" [disabled]="isCampaignReturnValueDisabled() || isCampaignReturnValueFixed()">
```

Comment:
Just to mention, if you want functions to look like variables, you can declare a function like this:
```
get isCampaignReturnValueDisabled() {
}
```
and then you can call them like a variable:
```
<fieldset class="contents" [disabled]="isCampaignReturnValueDisabled || isCampaignReturnValueFixed">
```

You don't need to do it here, but generally, you can do it :)

Reply:
Good to know, thanks!